vite.config.ts

```diff
@@ -23,47 +23,47 @@ const entryPoints = {
 		const name = item
 			.replace(/\/index\.(j|t)s/, '')
 			.replace('src/directives/', 'Directives/')
-		acc[name] = join(__dirname, item)
+		acc[name] = join(import.meta.dirname, item)
```

Comment:
supported since 20.11.0

Problem here: `__dirname` is not injected in playwright.config by vite so we need something more standard or we use the old hack with:
`const __dirname = dirname(fileURLToPath(import.meta.url));`

Reply:
But I think we can (as this is developer only) assume the nextcloud-vue developers have somewhat modern node, or what do you think?

Reply:
> supported since 20.11.0

Yes, and we require ^20.0.0

https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/package.json#L111

Reply:
> But I think we can (as this is developer only) assume the nextcloud-vue developers have somewhat modern node, or what do you think?

I'm also fine to adjust `package.json`